docker-compose.yml

```diff
@@ -84,7 +84,7 @@ services:
       - POSTGRES_DB=hyperswitch_db
 
   migration_runner:
-    image: rust:1.65
+    image: rust:1.69
```

Comment:
If you'd like to update to the latest Rust image as of today, that'd be 1.70.

Reply:
Thanks, Updated to latest rust and removed the ENV variable for sparse index.